qemu/tests/cfg/blockdev_inc_backup_non_persistent_bitmap.cfg

```diff
@@ -0,0 +1,35 @@
+- blockdev_inc_backup_non_persistent_bitmap:
+    only Linux
+    no libcurl, libssh
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_inc_backup_non_persistent_bitmap
+    virt_test_type = qemu
+    images += " data"
+    source_images = "data"
+    image_backup_chain_data = "base"
+    remove_image_data = yes
+    force_create_image_data = yes
+    storage_pools = default
+    storage_pool = default
+
+    image_size_data = 2G
+    image_size_base = 2G
+
+    image_format_data = qcow2
+    image_format_base = qcow2
+
+    image_name_data = data
+    image_name_base = base
+    rebase_mode = unsafe
+    check_bitmaps = 'bitmaps:'
+
+    variants:
```

Comment:
After our discussion last week, please help to update the cfg here.

Reply:
done